dotcom-rendering/src/components/SubMeta.tsx

```diff
@@ -62,6 +63,27 @@ const listStyleNone = css`
 		height: 0;
 		width: 0;
 	}
+	display: flex;
+	flex-wrap: wrap;
+	gap: 1.5rem 0.25rem;
+	background-image: repeating-linear-gradient(
+			to bottom,
+			${palette.background.subMeta} 0px,
+			${palette.background.subMeta} 36px,
+			transparent 36px,
+			transparent 37px,
+			${palette.background.subMeta} 37px,
+			${palette.background.subMeta} 46px
+		),
+		repeating-linear-gradient(
+			to right,
+			${neutral[86]} 0px,
+			${neutral[86]} 3px,
+			transparent 3px,
+			transparent 5px
+		);
```

Comment:
Looking good!

I think it's good to add a little explanation of how we arrived at these specific pixel values for future travellers — ideally an interactive demo 😁